src/SegmentedControl/SegmentedControl.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import lodash from "lodash";
+
+import "./SegmentedControl.less";
+
+/*
+  SegmentedControl selection. Shows several options
+  as defined in the props and allows the user to select
+  one of those options.
+*/
+export class SegmentedControl extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {selected: props.defaultValue};
+    this.onSelect = this.onSelect.bind(this);
+  }
+
+  onSelect(e) {
+    const value = e.target.getAttribute("value");
+    this.setState({selected: value});
+    if (this.props.onSelect !== undefined) {
+      this.props.onSelect(value);
+    }
+  }
+
+  render() {
+    const cssClass = SegmentedControl.cssClass;
+    const selectableItems = lodash.map(this.props.selectableItems, (name, value) => {
+      const isSelected = value === this.state.selected;
+      const selectedClass = isSelected ? "selected" : "";
+      return (
+        <span
+          className={`${cssClass.SELECTABLE_ITEM} ${selectedClass}`}
+          onClick={this.onSelect}
```

Comment:
nit: no need to bind in the constructor if you use an arrow function (preferred es6 pattern)
```js
onClick={e => this.onSelect(e)}
```